Features/AdminCommands/BanCommand.cs

```diff
@@ -77,6 +71,7 @@ await _databaseService.NewBanRecordAsync(
 
 
             await ctx.RespondAsync(embed: embed.Build());
+            await targetUser.SendMessageAsync(embed: embed.Build());
```

Comment:
Send the user a different embed, saying eg. "You have been banned from [server]" Here it is very important that the server name is included, maybe also the server ID in the footer. Also, the user does not need to know the user who was banned (because it's themselves duh) and who the mod was that banned them. That should stay staff-internal. Also how to appeal and stuff once that's done